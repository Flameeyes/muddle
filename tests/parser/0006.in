Let's see if embedded evaluation (and some odd spacing) works...

Hello, ${${/top/pronoun-name}}
${  "${/top/animal-name}"  }
