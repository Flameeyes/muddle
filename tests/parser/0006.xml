<?xml version="1.0" ?>

<top>
  <data>
    <pronoun>her</pronoun>
    <animal>dog</animal>
  </data>
  <pronoun-name>/top/data/pronoun</pronoun-name>
  <animal-name>/top/data/animal</animal-name>
</top>
