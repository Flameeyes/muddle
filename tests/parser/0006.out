Let's see if embedded evaluation (and some odd spacing) works...

Hello, her 
dog